src/plugins/auto/src/schedule.cpp

```diff
@@ -0,0 +1,307 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "schedule.hpp"
+#include "async_infer_request.hpp"
+
+namespace ov {
+namespace auto_plugin {
+thread_local WorkerInferRequest* Schedule::m_this_worker_inferrequest = nullptr;
+// TODO: revert to the plain variable (see header file), when we moved to the next CentOS 8.x in our support matrix
+thread_local const char* Schedule::m_this_preferred_devicename = "";
+
+void Schedule::launch(ScheduleContext::Ptr context) {
+    m_context = context;
+    m_log_tag = context->m_log_tag;
+    m_plugin = std::const_pointer_cast<Plugin>(std::dynamic_pointer_cast<const Plugin>(context->m_plugin));
+    LOG_INFO_TAG("scheduler starting");
+    init();
+}
+
+ISyncInferPtr Schedule::create_sync_infer_request() {
+    auto compiled_model = std::dynamic_pointer_cast<CompiledModel>(
+            m_context->m_compiled_model.lock());
+    SoAsyncInferRequest request_to_share_blobs_with;
```

Comment:
`request_to_share_blobs_with` => `request_to_share_tensors_with`
and in other places where we have `blob` in names